core/src/test/java/org/testcontainers/containers/output/ContainerLogsTest.java

```diff
@@ -0,0 +1,36 @@
+package org.testcontainers.containers.output;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.startupcheck.OneShotStartupCheckStrategy;
+
+import static org.rnorth.visibleassertions.VisibleAssertions.assertEquals;
+import static org.testcontainers.containers.output.OutputFrame.OutputType.STDERR;
+import static org.testcontainers.containers.output.OutputFrame.OutputType.STDOUT;
+
+public class ContainerLogsTest {
+
+    @Rule
```

Comment:
we will have to refactor the whole test if we add one more container to this test, I would use try-with-resources instead

Reply:
Hmm, I'm been looking at this but I'm not sure that I see why we'd add another container. 🤔 

Do we need to change this?

Reply:
I was thinking about a test where you call `getLogs` on a running container